src/tablib/formats/_ods.py

```diff
@@ -2,10 +2,13 @@
 """
 
 import numbers
+from datetime import date, datetime, time
```

Comment:
To avoid the ambiguity of the two `datetime`s, can we follow https://adamj.eu/tech/2019/09/12/how-i-import-pythons-datetime-module/ and import like:
```suggestion
import datetime as dt
```

Reply:
Here's a PR to refactor: https://github.com/jazzband/tablib/pull/569